### DISCUSSION TITLE
Wiki/Lore Enhancment

### DISCUSSION THREAD

**Discussion by OcelotWalrus** [OcelotWalrus/Cromha-Expansion-plugin#119] [Plugin Enhancment]
Here we will talk about enhancing the Lore as well as the Wiki. Any ideas are welcome!

**Comment by OcelotWalrus** [comment]
Hey @lumbar527, could you edit the [Lumbarian Empire](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/wiki/Lumbarian-Empire) page with using as a model the [Cromha Empire](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/wiki/Cromha-Empire) page. By the way, the Lumbarian Empire is a 2.5 tier faction.

**Reply by lumbar527** [reply]
It is? I forgot about tiers. I guess it's a weaker tier than the Cromha Empire but uses bigger ships.

Sure, though for quotes I'll just make a note of them being not yet made.

**Reply by OcelotWalrus** [reply]
> It is? I forgot about tiers. I guess it's a weaker tier than the Cromha Empire but uses bigger ships.

Yes. The Lumbarian Empire don't necessary uses bigger ships than the Cromha Empire. The Cromha Empire currently have the 2 biggest ship of the game/plugin.

**Reply by OcelotWalrus** [reply]
> It is? I forgot about tiers. I guess it's a weaker tier than the Cromha Empire but uses bigger ships.

Yes. The Lumbarian Empire don't necessary uses bigger ships than the Cromha Empire. The Cromha Empire currently have the 2 biggest ship of the game/plugin. By the way, bigger don't mean better. Bigger ship don't necessary have better outfits/stats and they are also easier to target.

**Reply by lumbar527** [reply]
At some point, I hope to create a supermassive cruiser, sort of like a Star Destroyer. It would be really big.

**Reply by OcelotWalrus** [reply]
I think the Lumbarian Cruisers designs a kind of cool. If you want I'll work on a bigger Cruiser than the others at this point of the development. It would be called the Lumbarian Fork-Class.

**Reply by OcelotWalrus** [reply]
We'll keep talking about that in Lumbarian Main discussion

**Comment by OcelotWalrus** [comment]
Just one thing @lumbar527 : You forgot about the Sun Drive and the Solar Core in the Lumbarian Outfits wiki page.

**Reply by lumbar527** [reply]
I just made the weapons, and I also haven't finished all of the pages for those. I'll get to other outfits after I finish the current ones, and also the Solar Core is a Empire Rebels outfit that is used by the Lumbarian Empire.

**Reply by OcelotWalrus** [reply]
Sure.
